app/src/main/java/com/kodeco/android/coordplot/components/PlotSurface.kt

```diff
@@ -0,0 +1,104 @@
+import androidx.compose.ui.unit.dp
+import com.kodeco.android.coordplot.R
+import com.kodeco.android.coordplot.ui.theme.CoordPlotTheme
+import com.kodeco.android.coordplot.utils.toIntPercentage
+
+@Composable
+fun PlotSurface(screenOrientation: Int) {
+    var xFloatValue: Float by rememberSaveable { mutableStateOf(0.5f) }
+    var yFloatValue: Float by rememberSaveable { mutableStateOf(0.5f) }
+
+    if (screenOrientation == Configuration.ORIENTATION_PORTRAIT) {
+        Column(
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.Center,
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(16.dp)
+        ) {
+            Map(xPercent = xFloatValue, yPercent = yFloatValue)
+            MapSlider(
+                floatValue = xFloatValue,
+                onValueChanged = { xFloatValue = it },
+                text = stringResource(
+                    id = R.string.slider_axis_value,
+                    stringResource(R.string.x_label),
+                    xFloatValue.toIntPercentage()
+                )
+            )
+            MapSlider(
+                floatValue = yFloatValue,
+                onValueChanged = { yFloatValue = it },
+                text = stringResource(
+                    id = R.string.slider_axis_value,
+                    stringResource(R.string.y_label),
+                    yFloatValue.toIntPercentage()
+                )
+            )
+        }
+    } else {
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.Center,
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(16.dp)
+        ) {
+            Map(xPercent = xFloatValue, yPercent = yFloatValue)
+            Column {
+                MapSlider(
+                    floatValue = xFloatValue,
+                    onValueChanged = { xFloatValue = it },
+                    text = stringResource(
+                        id = R.string.slider_axis_value,
+                        stringResource(R.string.x_label),
+                        xFloatValue.toIntPercentage()
+                    )
+                )
+                MapSlider(
+                    floatValue = yFloatValue,
+                    onValueChanged = { yFloatValue = it },
+                    text = stringResource(
+                        id = R.string.slider_axis_value,
+                        stringResource(R.string.y_label),
+                        yFloatValue.toIntPercentage()
+                    )
+                )
+            }
+        }
+    }
+}
+
+@Preview(showBackground = true)
+@Composable
+fun PlotSurfacePreviewPortrait() {
+    CoordPlotTheme {
+        PlotSurface(Configuration.ORIENTATION_PORTRAIT)
+    }
+}
+
+@Preview(showBackground = true, device = Devices.AUTOMOTIVE_1024p, widthDp = 864, heightDp = 432)
+@Composable
+fun PlotSurfacePreviewLandscape() {
```

Comment:
Good job adding previews for both portrait and landscape! Makes it easier to visualize.